tests/acceptance/io_services_tests.py

```diff
@@ -0,0 +1,1338 @@
+"""
+Tests of nbio_interface.AbstractIOServices adaptations
```

Comment:
One of the benefits of the refactoring is that we can now extensively test all the ioloop adapters generically using the same test code.